src/lexer/regexp.ts

```diff
@@ -59,13 +59,13 @@ export function scanRegularExpression(parser: ParserState, context: Context): To
   const bodyEnd = parser.index - 1;
 
   const enum RegexFlags {
-    Empty = 0b00000,
-    IgnoreCase = 0b00001,
-    Global = 0b00010,
-    Multiline = 0b00100,
-    Unicode = 0b10000,
-    Sticky = 0b01000,
-    DotAll = 0b1100
```

Comment:
I got no idea why the previous bit-mask for DotAll 0b1100 is an union of Sticky 0b01000 and Multiline 0b00100.
It doesn't make any sense.